linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithDnf.java

```diff
@@ -43,25 +43,28 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
 		 * ubi8 has DNF installed
 		 */
 
-		String containerRegistry = "";
+		String quayIoDefaultRegistry = "quay.io/";
+		String dockerDefaultRegistry = "";
 
 		if (System.getenv("containerRegistry") == null) {
-			System.out.println("Using docker.io as the default container registry");
+			System.out.println("Using docker.io and quay.io as the default container registry");
 		} else {
-			containerRegistry = System.getenv("containerRegistry");
-			System.out.println("Using container registry: " + containerRegistry);
+			quayIoDefaultRegistry = dockerDefaultRegistry = System.getenv("containerRegistry");
+			System.out.println("Using container registry: " + dockerDefaultRegistry);
 		}
 
 		return Stream.of(
-			Arguments.of(containerRegistry + "rockylinux", "8"),
-			Arguments.of(containerRegistry + "fedora", "35"),
-			Arguments.of(containerRegistry + "fedora", "36"),
-			Arguments.of(containerRegistry + "fedora", "37"),
-			Arguments.of(containerRegistry + "fedora", "38"),
-			Arguments.of(containerRegistry + "fedora", "39"),
-			Arguments.of(containerRegistry + "redhat/ubi8", "latest"),
-			Arguments.of(containerRegistry + "redhat/ubi9", "latest"),
-			Arguments.of(containerRegistry + "oraclelinux", "8")
+			Arguments.of(quayIoDefaultRegistry + "centos/centos", "9"),
+			Arguments.of(quayIoDefaultRegistry + "centos/centos", "10"),
+			Arguments.of(dockerDefaultRegistry + "rockylinux", "8"),
```

Comment:
Just for  curiosity, why rocky 8, and not centos 8?

Reply:
rockylinux was already there, although I did som